libs/auth/scripts/src/generate_machine_label_qr_code_contents.ts

```diff
@@ -1,134 +0,0 @@
-import path from 'node:path';
```

Comment:
Looks like this and a few other related changes in libs/auth for https://github.com/votingworks/vxsuite/pull/5768 accidentally got lost, too - will bring them back!